apps/web/app/globals.css

```
@@ -120,3 +120,26 @@
   }
 }
 
+@media print {
+  @page {
+    size: A4;
```

Stylelint (17.14.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)